[Service Now] Update Mid Server paragraph in Service Graph setup (#468)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/integrations/service-now/service-graph/service-graph-setup.mdx

![ServiceNow Service Graph Connector HTTP Connection configuration screen. A Terraform Enterprise URL may be provided in the Connection URL field, the saved by clicking the Update button](/img/docs/service-now-service-graph-tfconn.png)

> If you are using **Terraform Enterprise**, a **MID server** is **required** to faciliate communication between ServiceNow and the TFE API. If you are using **HCP Terraform**, the **MID server** is **optional**. 

Reviewer: ```suggestion
If you are using Terraform Enterprise, ServiceNow requires a Management, Instrumentation, and Discovery (MID) server to communicate with the Terraform Enterprise API. If you are using HCP Terraform, a MID server is optional.
```

To use a MID server:

> 1. Check **Use MID server** box

Reviewer: ```suggestion
1. Select the **Use MID server** checkbox.
```

> 1. Change **MID Selection** dropdown to "Specific MID sever"

Reviewer: ```suggestion
1. Change **MID Selection** dropdown to "Specific MID sever".
```

> 2. Select your previously configured and validated MID server

Reviewer: ```suggestion
2. Select your previously configured and validated MID server.
```
